src/Symfony/Bundle/FrameworkBundle/Resources/config/asset_mapper.php

```diff
@@ -162,6 +163,12 @@
                 abstract_arg('vendor directory'),
             ])
 
+        ->set('asset_mapper.importmap.version_checker', ImportMapVersionChecker::class)
```

Comment:
We cannot rely on a class in the `Tests` namespace, this should be moved elsewhere.

Reply:
Thank you - accidental, but I've done that twice lately 🙃